ui/imports/shared/views/AssetsView.qml

```diff
@@ -19,6 +19,7 @@ Item {
         id: assetListView
         anchors.fill: parent
         model: account.assets
+        objectName: "assetList"
```

Comment:
Maybe set a more descriptive name like `assetViewStatusListView` to avoid name clashes